tensorflow/python/ops/image_ops_impl.py

```diff
@@ -643,15 +643,17 @@ def transpose(image, name=None):
 
   Usage Example:
   
-  >>> image = tf.random.normal(shape=(3, 2, 3))  
+  >>> image = [[[0.0, 0.1, -1.0 ],[0.0,  1.0,  0.1]], [[-1.0,  1.0,  0.1 ], [ 0.1, -2.0, -0.0]],[[-1.0, 0.1,  1.0 ], [0.0, -1.0 , 0.1 ]]]
```

Comment:
Can you use an array like this:

image = [[1, 2, 3],
                 [4, 5, 6],
                 [7, 8, 9]]

and then show the output for this?

Also, remember to use `...` for line continuation. See the guide and please test it out locally so that you will see the errors.

Reply:
Also, see the docstrings for `tf.concat` as an example.

Reply:
the array you gave raises two value errors
ValueError: Shape (3, 3) must have rank at least 3
ValueError: 'image' must be at least three-dimensional.

Reply:
I have seen it let me change the arrays i used

Reply:
I have made the changes please see if there is any other thing to be done before this PR can be merged.